README.md

```diff
@@ -0,0 +1,3 @@
+# Massa Smart Contracts Runtime
+
+This is a VM (basically a wrapper around [Wasmer](https://wasmer.io/)) that lets run WASM SC generated with AssemblyScript (and using Massa customs ABIs).
```

Comment:
Link to the tutorial on the other folder ?

Reply:
Why not! What is the exact link you would like to see here?

Reply:
The tutorial that was in the PR but is now in RTD. But it's not necessary.